entity_parser_key_parser_test.go

```diff
@@ -526,8 +536,49 @@ func TestEntityParse(t *testing.T) {
 					},
 				},
 			},
+			ETL:   EtlOff,
 			Error: nil,
 		},
+		{
+			Tag:       "name=jj primaryKey=ok, etl=on",
```

Comment:
are we choosing 'on/off' over 'enabled/disabled'? just confirm

Reply:
yes. on/off seems more concise and accurate to me